wmn-data.json

```diff
@@ -6537,7 +6484,7 @@
         "uri_check" : "https://zmarsa.com/uzytkownik/{account}/glowna/",

```

Comment:
This `uri_check` redirects me to `https://zmarsa.com/uzytkownik/{account}`. Can you redo the `e_string` and `m_string` from that page? I think the following will work:
`e_string` = `<title>Użytkownik`
`m_string` = `<title>Error 404`